src/lib/transferExpiryMonitor.js

```diff
@@ -54,7 +55,7 @@ class TransferExpiryMonitor {
 
         yield _this.notificationBroadcaster.sendNotifications(transfer, transaction)
       }
-    })
+    }, 10) // retries if database is busy
```

Comment:
Why 10 times? Constants like this should be declared at the top of the file so they're easier to find later